test/mocha.opts

```diff
@@ -1 +0,0 @@
---ui tdd
```

Comment:
@wuweiss, how should libraries that depend on sanctuary-scripts specify Mocha options after this change? Currently we use [__mocha.opts__][1].


[1]: https://github.com/sanctuary-js/sanctuary/blob/v3.1.0/test/mocha.opts

Reply:
Ah, something else I missed is that the PR title states that mocha.opts is deprecated. Perhaps sanctuary-scripts should switch to advertising whatever approach replaces it.